docs/offboarding.rst

```diff
@@ -250,16 +247,7 @@ Return to the Secure Viewing Station
 
 #. **Do not delete your old submission key!** You'll want to maintain it on
    the *SVS* so that you can still decrypt old submissions that were made
-   before you changed keys. If you like, you can revoke the key by
-   selecting the key in the **Passwords and Keys** application, opening the
-   **Details** tab, highlighting the first key entry and clicking
-   **Revoke**. This also makes local-only changes and does not stop you or
-   anyone else from using the key, but it is a reminder that your key has
-   changed.
-
```

Comment:
The way to do this in Kleopatra appears to be to generate a revocation certificate, edit it with a text editor to remove the colon inserted for security reasons, and then import it into Kleopatra.

Reply:
@eloquence Thanks! I'm trying to document this process, but I can't seem to get Kleopatra to actually save the revocation certificate. Have tried saving in various places in the filesystem, as well as the Persistent storage, on multiple Tails USBs. Just to note here, the version of Kleopatra on the latest Tails is 3.1.11.

After digging into this a bit further, I think I'm running into this bug: https://bugs.kde.org/show_bug.cgi?id=384013

When you tested this, was it using a local copy of Kleopatra, or was it booted to a Tails disk? Do you happen to know the version? 

In the meantime, I'm going to try and come up with instructions using the CLI to avoid these bugs.

Reply:
I just pushed documentation for a process that uses the CLI to create the revocation certificate to work around the bug mentioned above. Please review and let me know if you think this is a good method to avoid the issues I ran into earlier. Thanks!

Reply:
I tested it in Tails, and I think I know why we saw different behavior. I started `kleopatra` from the terminal first, where I see output related to the revocation, and it's saved correctly. If you start the application without a TTY attached to it, I am also unable to save it (it may never successfully shell out to `gpg`). I think CLI instructions are indeed the way to go, will review.